### ISSUE TITLE
Swift 3 branch

### ISSUE THREAD

**Issue by lfarah** [alvinvarghese/EasyIAPs#4]
Can we create a new **swift3** branch so we can start porting **EasyIAPs** to Swift 3.

**Comment by alvinvarghese** [comment]
Yeah, I haven't thought about it. Will do it as soon as possible.
